packages/icons/src/index.ts

```diff
@@ -1,44 +1,294 @@
+import { default as CaretRight } from "./components/CaretRight";
+import { default as CaretUp } from "./components/CaretUp";
+import { default as Caution } from "./components/Caution";
+import { default as CheckCircleFilled } from "./components/CheckCircleFilled";
+import { default as CheckCircle } from "./components/CheckCircle";
+import { default as Check } from "./components/Check";
+import { default as HelpCircleFilled } from "./components/HelpCircleFilled";
+import { default as Copy } from "./components/Copy";
+import { default as Download } from "./components/Download";
+import { default as EllipsisVertical } from "./components/EllipsisVertical";
+import { default as Ellipsis } from "./components/Ellipsis";
+import { default as ErrorCircleFilled } from "./components/ErrorCircleFilled";
+import { default as ErrorCircle } from "./components/ErrorCircle";
+import { default as EyeOff } from "./components/EyeOff";
+import { default as Eye } from "./components/Eye";
+import { default as Gear } from "./components/Gear";
+import { default as InfoCircleFilled } from "./components/InfoCircleFilled";
+import { default as InfoCircle } from "./components/InfoCircle";
+import { default as Invalid } from "./components/Invalid";
+import { default as List } from "./components/List";
+import { default as LockFilled } from "./components/LockFilled";
+import { default as Lock } from "./components/Lock";
+import { default as MinusCircle } from "./components/MinusCircle";
+import { default as Minus } from "./components/Minus";
+import { default as Open } from "./components/Open";
+import { default as Org } from "./components/Org";
+import { default as Pencil } from "./components/Pencil";
+import { default as PlusCircle } from "./components/PlusCircle";
+import { default as Plus } from "./components/Plus";
+import { default as Search } from "./components/Search";
+import { default as Stack } from "./components/Stack";
+import { default as Star } from "./components/Star";
+import { default as Terminal } from "./components/Terminal";
+import { default as Time } from "./components/Time";
+import { default as User } from "./components/User";
+import { default as World } from "./components/World";
+
+// Pictograms
+import { default as Add } from "./components/Add";
+import { default as Cluster } from "./components/Cluster";
+import { default as Email } from "./components/Email";
+import { default as Globe } from "./components/Globe";
+import { default as Location } from "./components/Location";
+import { default as Monitoring } from "./components/Monitoring";
+
+// Illustrations
+import { default as MagnifyingGlass } from "./components/MagnifyingGlass";
+import { default as Menu } from "./components/Menu";
+import { default as Nodes } from "./components/Nodes";
+import { default as NotFound } from "./components/NotFound";
+import { default as SleepyMoon } from "./components/SleepyMoon";
+
+// 3rd Party Icons
```

Comment:
I don't consider this blocking, but it's worth noting that these icons are only used in CC Console. Having a separate icon set in CC Console for CC Console specific icons would make it easier for us to modify or add to the existing set. If they are in the UI repository, then we will need to update multiple packages and bump the versions of each in order to add anything. It's a lot of friction.

I believe that 3rd party icons, cards, and flags are only used in CC Console.

Reply:
While it is true that currently the majority of these icons are only used in the CockroachCloud console, they are a universally consistent part of the design system. So when we use flags or credit  cards or third party logos, we are using consistent graphics. I'm not sure if we have Cloud console _specific_ icons, but if we do please let me know.

I am sympathetic to the potentially increased friction of updating svgs outside of an application repository, but historically there have been very few changes to these icons once they exists. If the concern is that having icons centralized will hinder development at the application level, I would point out that engineers are free to do what they need to do to meet their goals. If that means using local SVGs or local icon components over centralized ones, we should discuss that as those needs arise. But the goal of this repository remains to provide portable resources that are consistent with Design System specification and I believe these icon sets are included in that.